app/ui/design-system/src/lib/Pages/ConceptsPage/index.tsx

```diff
@@ -0,0 +1,150 @@
+import { ButtonLink } from '../../Components/Button';
+import {
+  ContentNavigation,
+  ContentNavigationProps,
+} from '../../Components/ContentNavigation';
+import { EventCard, EventCardProps } from '../../Components/EventCard';
+import {
+  FeatureLinkBlock,
+  FeatureLinkBlockProps,
+} from '../../Components/FeatureLinkBlock';
+import {
+  LandingHeader,
+  LandingHeaderProps,
+} from '../../Components/LandingHeader';
+import { ToolCard, ToolCardProps } from '../../Components/ToolCard';
+
+export interface ConceptsPageProps {
+  landingHeaderItems: LandingHeaderProps;
+  featureLinkBlockItems: [FeatureLinkBlockProps, FeatureLinkBlockProps];
+  toolCardItems: [
+    ToolCardProps,
+    ToolCardProps,
+    ToolCardProps,
+    ToolCardProps,
+    ToolCardProps,
+    ToolCardProps
+  ];
+  contentNavigationItems: [
+    ContentNavigationProps,
+    ContentNavigationProps,
+    ContentNavigationProps
+  ];
+  eventCardItems: EventCardProps;
+}
+
+export function ConceptsPage({
+  landingHeaderItems,
+  featureLinkBlockItems,
+  toolCardItems,
+  contentNavigationItems,
+  eventCardItems,
+}: ConceptsPageProps) {
+  return (
+    <div className="bg-primary-gray-50 dark:bg-black">
+      <div className="pb-10 md:border-b md:border-primary-gray-100 md:pb-28">
+        <LandingHeader
+          title={landingHeaderItems.title}
+          buttonText={landingHeaderItems.buttonText}
+          buttonUrl={landingHeaderItems.buttonUrl}
+          gradient={landingHeaderItems.gradient}
+          callout={landingHeaderItems.callout}
+          description={landingHeaderItems.description}
+        />
+      </div>
+
+      <div className="pb-32 md:border-b md:border-primary-gray-100 md:pb-28 md:pt-20">
+        <div className="container grid grid-cols-1 gap-y-7">
+          {featureLinkBlockItems.map((featureLinkBlock, i) => (
+            <FeatureLinkBlock
+              key={i}
+              title={featureLinkBlock.title}
+              description={featureLinkBlock.description}
+              links={featureLinkBlock.links}
+              icon={featureLinkBlock.icon}
+              ctaLink={featureLinkBlock.ctaLink}
+              ctaText={featureLinkBlock.ctaText}
+            />
+          ))}
+        </div>
+      </div>
+
+      <div className="pb-40 md:border-b md:border-primary-gray-100 md:pt-20 md:pb-28">
+        <div className="container">
+          <div className="flex justify-between md:pb-11">
+            <div>
+              <h4 className="text-h4 md:text-h2 pb-4 text-2xl md:pb-2">
```

Comment:
The tools section is the same as the one on the tools page, and very similar to the ToolsAndConcepts. What do you think about reusing the ToolsAndConcepts component in both places?

It can be modified to only show the tabs if both tools and concepts are passed as props, and take in custom props for the title/subtitle

Reply:
@neilZon looks like @aureentuluvacal already updated ToolsAndConcepts in #149 so it can be reused